From b714fdebc138e15ef90202076e0c89df85868e52 Mon Sep 17 00:00:00 2001
From: Tom Hochstein <tom.hochstein@nxp.com>
Date: Tue, 6 Apr 2021 12:15:40 -0500
Subject: [PATCH] RenderToTexture: Fix incorrect G2D dependency

```
| In file included from /opt2/hardknott/build/tmp/work/cortexa53-crypto-mx8m-poky-linux/imx-gpu-sdk/5.7.0-r0/git/DemoApps/GLES3/RenderToTexture/source/RenderToTexture.cpp:32:
| /opt2/hardknott/build/tmp/work/cortexa53-crypto-mx8m-poky-linux/imx-gpu-sdk/5.7.0-r0/git/DemoApps/GLES3/RenderToTexture/source/RenderToTexture.hpp:44:10: fatal error: g2d.h: No such file or directory
|    44 | #include <g2d.h>
|       |          ^~~~~~~
| compilation terminated.
```

Signed-off-by: Tom Hochstein <tom.hochstein@nxp.com>
---
 DemoApps/GLES3/RenderToTexture/Fsl.gen | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DemoApps/GLES3/RenderToTexture/Fsl.gen b/DemoApps/GLES3/RenderToTexture/Fsl.gen
index 0f075e5a..91560c5a 100644
--- a/DemoApps/GLES3/RenderToTexture/Fsl.gen
+++ b/DemoApps/GLES3/RenderToTexture/Fsl.gen
@@ -6,7 +6,7 @@
     <Dependency Name="FslSimpleUI.App"/>
     <Dependency Name="FslSimpleUI.Theme.Basic"/>
     <Platform Name="Yocto" Supported="true">
-      <ExternalDependency Name="g2d" Type="StaticLib"/>
+      <Dependency Name="G2D"/>
       <Requirement Name="GL_VIV_direct_texture" Type="extension" Extends="OpenGLES3"/>
     </Platform>    
   </Executable>
-- 
2.17.1

